homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -156,10 +141,9 @@ def state(self) -> str | None:
             state = STATE_ALARM_TRIGGERED
             attr["triggered_source"] = "Carbon Monoxide"
 
-        self._state = state
         self._attr_extra_state_attributes = attr
```

Comment:
Also consider if this should be moved in a future pr so that `state` doesn't have side effects

Reply:
That was also one of my goals